website/content/en/releases/13.1R/relnotes.adoc

```diff
@@ -108,6 +118,7 @@ This section covers changes to kernel configurations, system tuning, and system
 [[kernel-general]]
 === General Kernel Changes
 
+…
```

Comment:
_Stack address randomization reimplementation_ – <https://forums.freebsd.org/posts/557777>

Reply:
<https://forums.freebsd.org/posts/567106> leads indirectly to <https://github.com/freebsd/freebsd-src/commit/5fa005e91560785dad5183db080209447afde3c2>